src/main/resources/db/migration/v0.16.0.5__agr_curation_api.sql

```diff
@@ -0,0 +1,33 @@
+DELETE FROM reference_crossreference
+	WHERE reference_curie IN (
+		SELECT curie FROM informationcontententity
+			WHERE obsolete = true
+	);
+	
+DELETE FROM reference
+	WHERE curie IN (
+		SELECT curie FROM informationcontententity
+			WHERE obsolete = true
+	);
+	
+DELETE FROM slotannotation_informationcontententity
+	WHERE evidence_curie IN (
+		SELECT curie FROM informationcontententity
+			WHERE obsolete = true
+	);
+	
+DELETE FROM informationcontententity
+	WHERE obsolete = true;
+	
+DELETE FROM ontologyterm_crossreference;
+	
+DELETE FROM genomicentity_crossreference;
```

Comment:
@markquintontulloch I am guessing you want to delete from vs drop so that they can be reloaded?

Reply:
exactly - we're not currently loading xrefs for genomic entities (haven't been requested from DQMs yet, although some are submitting them) but there are loads that got loaded way back before we refactored all the loading and validation stuff